APE0.rst

```diff
@@ -0,0 +1,393 @@
+
+When delegating its authority to create roles or make appointments, the
+Coordination Committee must solicit community feedback for no less than two
+weeks.
+
+Making Decisions
+^^^^^^^^^^^^^^^^
+To use its authority, the Coordination Committee must first attempt to reach
+a consensus. If a consensus cannot be reached the Committee votes. Every
+Coordination Committee member must either vote or explicitly abstain. Members
+with a conflict of interest on a particular vote must abstain. The final
+decision regarding what constitutes a conflict of interest is determined by the
+Ombudsperson in accordance with the conflict of interest policy. Passing
+decisions require a strict majority of non-abstaining Coordination Committee
+members.
+
+Whenever possible, the Coordination Committee's deliberations shall be held in
+public in a manner that permits discussion and comments from the community.
+Final decisions made by the Coordination Committee will be published in a
+publicly visible location.
+
+Electing the Coordination Committee
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A Coordination Committee election is held annually to fill any vacant seats.
+Notice of the election must be posted via the Project’s standard communication
+channels (reaching all active members) at least one month before the election
+begins. The election consists of two phases, each lasting two weeks:
+
+* Phase 1, Nominations: Candidates will be nominated and confirm their interest
+  in serving. Candidates can be nominated by anyone, including themselves, and
+  do not have to be `Voting Members <votingmembers>`_ to be nominated.
+  Candidates should provide a brief statement in support of their candidacy.
+* Phase 2, Voting: Each `Voting Member <votingmembers>`_ can vote for zero or
+  more of the candidates, up to the number of candidates. Voting is performed
+  anonymously. Candidates are ranked by the total number of votes they receive.
+  If a tie occurs, it may be resolved by mutual agreement among the candidates,
+  or else the winner will be chosen at random.
+
+The election process is managed by a non-voting Returns Officer nominated by
+the outgoing Coordination Committee, although the process (specific choice of
+web site, etc) can be set up by the Coordination Committee or their delegates.
+For the initial election, the Returns Officer will be nominated by the NumFOCUS
+Executive Officer. Elections will be carried out every year, provided that at
+least one of the Coordination Committee members has reached the end of their
+term or decided to step down.
+
+Term
+^^^^
+Each Coordination Committee member's term runs for three years from when
+the election results are finalized, except when the election was to fill a
+vacancy created partway through a term. In that case, the newly-elected member’s
+term runs for the remainder of the term of the person they are replacing. There
+is no limit to the number of terms that a single individual can be elected for.
```

Comment:
The Coordination Committee discussed this and we suggest the following:
```suggestion
Each Coordination Committee member's term runs for three years from when
the election results are finalized. There is no limit to the number of terms that
a single individual can be elected for.

In the case of a vacancy partway through a term, a by-election will be held. 
The term of the newly-elected member runs for the remainder of the term of
the member they are replacing. 
```
this aligns with @bsipocz suggestion of calling it a by-election.

We also discussed @perrygreenfield's point and agreed that we think that an approach like that could work but creates some added complexity, and would open the door for a member to have a 4.5 year term (which would require more rules to prevent).  Given the Working Group was particularly motivated by keeping the election process relatively simple, we are thinking it's better to leave this as it stands. So we think instead to leave it to the voting members to judge if a by-election is needed if it's already close to the end of the term.